### PR TITLE
added manual trigger endpoint for removing expired provisional bookings

### PR DESCRIPTION
**Pull request by jrowett**

### CODE REVIEW

File: src/api/Nhs.Appointments.Api/Validators/ConfirmBookingRequestValidator.cs [hunk 1]
```diff
@@ -10,6 +10,8 @@ public ConfirmBookingRequestValidator()
     {
         RuleFor(x => x.bookingReference)
             .NotEmpty().WithMessage("Provide a valid booking reference");
+        RuleFor(x => x.contactDetails)
```
Comment (vicr1) [inline src/api/Nhs.Appointments.Api/Validators/ConfirmBookingRequestValidator.cs:13]: It should be possible to confirm without providing contact details

**Review by vicr1 — CHANGES_REQUESTED** [review]

**Review by kicr1 — CHANGES_REQUESTED** [review]
I think we need an integration test for this as it ensures that the cleardown process is working correctly,

File: src/api/Nhs.Appointments.Api/Notifications/template-bookingreminder-rsv-email.txt [hunk 2]
```diff
@@ -22,6 +22,7 @@ Date and time: ((date))
 
 # If you need to change or cancel your appointment:
 
-* visit https://www.nhs.uk/book-rsv
+* Visit https://www.nhs.uk/book-rsv
+* If you cant get online, call 119
```
Comment (sambiramairelogic) [inline src/api/Nhs.Appointments.Api/Notifications/template-bookingreminder-rsv-email.txt:26]: We're missing an apostrophe in can't - could make it cannot if the apostrophe causes serialisation issues?

Comment (jrowett) [reply src/api/Nhs.Appointments.Api/Notifications/template-bookingreminder-rsv-email.txt:26]: oof!

File: tests/Nhs.Appointments.Api.Integration/Scenarios/Booking/ConfirmBookingFeatureSteps.cs [hunk 3]
```diff
@@ -71,5 +77,12 @@ public async Task AssertBookingContactDetails(Gherkin.Ast.DataTable dataTable)
         var actualBooking = await Client.GetContainer("appts", "booking_data").ReadItemAsync<BookingDocument>(bookingReference, new Microsoft.Azure.Cosmos.PartitionKey(siteId));
         actualBooking.Resource.ContactDetails.Should().BeEquivalentTo(expectedContactDetails);
     }
+
+    [And("the number of bookings removed should be in the response")]
```
Comment (kicr1) [inline tests/Nhs.Appointments.Api.Integration/Scenarios/Booking/ConfirmBookingFeatureSteps.cs:81]: Sorry to be a pain, but is there anyway that you could assert in the database that the expected records have actually been removed?

**Review by kicr1 — APPROVED** [review]

**Pull request merged**